protocol/protocol.tex

```diff
@@ -3388,17 +3389,6 @@
 \sapling{There is no equivalent of interstitial \treestates for \Sapling\nufive{ or
 for \Orchard}.}
 
-\nufive{
-\vspace{1ex}
-$\MerkleCRH{Orchard}$ can produce $\bot$ as output (with insignificant probability).
-If either input is $\bot$, this is propagated to the output, and so if any \merkleNode
-of a \noteCommitmentTree is $\bot$, then the \merkleRoot of that tree will be $\bot$.
-
-\vspace{-1ex}
-\consensusrule{The \merkleRoot of the \Orchard \noteCommitmentTree \MUSTNOT be $\bot$
-in any (intermediate or output) \treestate created by a \block.}
```

Comment:
Note that after this change, the Orchard Merkle root in a final treestate (not just an intermediate treestate) is allowed to be 0. There's no advantage in making a distinction between intermediate and final roots.